website/src/types/Organism.ts

```diff
@@ -67,6 +67,7 @@ export const organismConfig = {
 };
 export const allOrganisms = Object.keys(organismConfig) as Organism[];
 export type Organism = keyof typeof organismConfig;
+export type OrganismsWithViewKey<T, ViewKey extends string> = T extends `${infer O}.${ViewKey}` ? O : never;
```

Comment:
Let's put this in `routing.ts` (next to the definition of `OrganismViewKey`)

Reply:
I think that it would be more consistent to write `Organism` as singular.

```suggestion
export type OrganismWithViewKey<T, ViewKey extends string> = T extends `${infer O}.${ViewKey}` ? O : never;
```

Reply:
Done :-)